### DISCUSSION TITLE
Creating new ER Diagram for project different from General

### DISCUSSION THREAD

**Discussion by viacheslav-sabadash** [dbeaver/dbeaver#16859] [General]
DBeaver Version 22.1.0.202206051253 (MacOS Monterey 12.4)

I created new project and move one connection to it from **General** project.
When I try create **ER Diagram** in this new project I see in **Diagram Creation Wizard** window only list of connections which i  leave in **General** project, but without this one connection which was attached to current new project which need to create diagram.

This is a bug or I do something wrong?

**Comment by LonwoLonwo** [comment]
Hello @viacheslav-sabadash 

Did you try to set first your project up as active and then create an ERD diagram?

![2022-06-22 17_24_14-](https://user-images.githubusercontent.com/45152336/175053886-9b376789-03e7-48cf-a753-c4555ba302f9.png)

**Reply by viacheslav-sabadash** [reply]
Hello @LonwoLonwo 
Thanks! Looks like this is a solution in my case.
